### ISSUE TITLE
Translating the error message from unsuccesful login

### ISSUE THREAD

**Issue by apertt** [signalpoint/DrupalGap#926]
Hi,

when user tries to login with wrong username or password, the corresponding error message is shown as an alert. I am not able to translate that string. It's in my locale language file and other strings are translated as should, but not this one.

I tried to search the code to see where it's been set, but was unsuccessful at that too. So should the error string be translatable in the first place?

**Comment by signalpoint** [comment]
@apertt After digging into the code a bit, the error message should be properly translated here: https://github.com/signalpoint/DrupalGap/blob/7.x-1.x/src/modules/user/user.js#L227

The error message itself is actually the real form error message generated by Drupal. It's possible the message may be wrapped in quotes or brackets when delivered via JSON and then the translator misses it. Will you please send a screen shot of what error message you see?

**Comment by apertt** [comment]
@signalpoint Thanks for locating the code! Screenshot attached. In the console I can see the error message also and it is indeed in brackets (or is it an array with one item).
![screenshot_20170427-064639](https://cloud.githubusercontent.com/assets/2913465/25468023/5c8aa990-2b1b-11e7-8353-03ec1d7efb7a.png)

**Comment by apertt** [comment]
Well, I managed to solve the problem by translating the string via the actual Drupal site the app is hooked on. Still not sure why it was not translated in the app, but the end result is the same now.

Before that I messed with the code a bit to see what was going on, even commenting out the actual alert in line 229, but none of my changes took effect. I have caching off and debug on, so I'm not sure why.
